zookeeper-server/src/main/java/org/apache/zookeeper/server/ZooKeeperServer.java

```diff
@@ -1627,6 +1628,22 @@ public void processPacket(ServerCnxn cnxn, ByteBuffer incomingBuffer) throws IOE
         }
     }
 
+    private static boolean isSaslSuperUser(String id) {
+        String superUser = System.getProperty(SASL_SUPER_USER);
+
+        if (superUser == null || id == null || id.isEmpty()) {
+            return false;
+        }
+
+        for (String superId : superUser.split(",")) {
```

Comment:
In order to be 100% compatible....what about using a list of properties 
- zookeeper.superUser=foo
- zookeeper.superUser.1=foo2
- zookeeper.superUser.2=foo3
- zookeeper.superUser.3=foo4

Reply:
I like this! (Particularly since there is a precedent.  Sorta.)